docs/ai/conceptual/evaluation-libraries.md

```diff
@@ -33,8 +39,19 @@ The evaluation libraries were built in collaboration with data science researche
 | Fluency      | Evaluates grammatical accuracy, vocabulary range, sentence complexity, and overall readability| <xref:Microsoft.Extensions.AI.Evaluation.Quality.FluencyEvaluator> |
 | Coherence    | Evaluates the logical and orderly presentation of ideas | <xref:Microsoft.Extensions.AI.Evaluation.Quality.CoherenceEvaluator> |
 | Equivalence  | Evaluates the similarity between the generated text and its ground truth with respect to a query | <xref:Microsoft.Extensions.AI.Evaluation.Quality.EquivalenceEvaluator> |
-| Groundedness | Evaluates how well a generated response aligns with the given context | <xref:Microsoft.Extensions.AI.Evaluation.Quality.GroundednessEvaluator><br />`GroundednessProEvaluator` |
-| Protected material | Evaluates response for the presence of protected material | `ProtectedMaterialEvaluator` |
+| Groundedness | Evaluates how well a generated response aligns with the given context | <xref:Microsoft.Extensions.AI.Evaluation.Quality.GroundednessEvaluator> |
+| Relevance (RTC), Truth (RTC), and Completeness (RTC) | Evaluates how relevant, truthful, and complete a response is | <xref:Microsoft.Extensions.AI.Evaluation.Quality.RelevanceTruthAndCompletenessEvaluator>† |
+
+† This evaluator is marked [experimental](../../fundamentals/syslib-diagnostics/experimental-overview.md).
+
+### Safety evaluators
+
+Safety evaluators check for presence of harmful, inappropriate, or unsafe content in a response. They rely on the Azure AI Foundry Evaluation service, which uses a model that's fine tuned to perform evaluations.
+
+| Metric             | Description                                                           | Evaluator type               |
+|--------------------|-----------------------------------------------------------------------|------------------------------|
+| Groundedness       | Evaluates how well a generated response aligns with the given context | `GroundednessProEvaluator`   |
```

Comment:
Can we link to the docs for the evaluators in the `Evaluator type` column like you did for the quality evaluators above?

Reply:
Yes, let me update the API docs first so I can link to them.